k8s/openebs-pre-release-features.yaml

```diff
@@ -687,21 +687,21 @@ spec:
       replicas: 1
       selector:
         matchLabels:
-          {{- if eq $isMonitor "true" }}
-          monitoring: volume_exporter_prometheus
-          {{- end}}
+          app: cstor-volume-manager
           openebs.io/target: cstor-target
           openebs.io/persistent-volume: {{ .Volume.owner }}
-          app: cstor-volume-manager
       template:
         metadata:
           labels:
             {{- if eq $isMonitor "true" }}
             monitoring: volume_exporter_prometheus
+            openebs_pv: {{ .Volume.owner }}
             {{- end}}
+            app: cstor-volume-manager
             openebs.io/target: cstor-target
             openebs.io/persistent-volume: {{ .Volume.owner }}
-            app: cstor-volume-manager
+            openebs.io/persistent-volume-claim: {{ .Volume.pvc }}
```

Comment:
Why same two labels with same value?

Reply:
The below `pvc` is kept for tools that can't sustain a `.` in the label key - like the prometheus source label config. we might remove that `pvc` however in the future if no-one is reading it.